ConsoleApp1/ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+
+namespace Homework
+{
+    public class Homework
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine(MaxV(1,2));
+                Console.WriteLine(MaxV(2,3,5,6,7));
+        }
+
+        //створити метод Max приймає від 2 до 5 аргументів та повертає max відповідно значення серед цих аргументів
+        public static int MaxV(int v1, int v2, int v3 = -2147483648, int v4 = -2147483648, int v5 = -2147483648)
```

Comment:
cool, not critical, but you can use constant int.MinValue and int.MaxValue instead, just to let you know